gitpod-io/gitpod
# enable IntelliJ and GoLand indexing in prebuilds

## Description


Enable IntellIJ and GoLand indexing in prebuilds using experimental feature in .gitpod.yml.

## Related Issue(s)

Fixes #

## How to test


- A prebuild should be successful: https://gitpod.io/t/gitpod/gitpod/ef0b8b89-e367-4289-a52a-33feb0f5f8d9
- Enable stable GoLand in preferences: https://gitpod.io/preferences
- Start a workspace for this PR: https://gitpod.io#https://github.com/gitpod-io/gitpod/pull/9561
  - Verify that indexing and module resolution is fast
- Enable stable IntelliIJ IDEA in preferences: https://gitpod.io/preferences
- Start a workspace for this PR: https://gitpod.io#https://github.com/gitpod-io/gitpod/pull/9561
  - Verify that indexing and module resolution is fast

## Release Notes

```release-note
NONE
```

## Documentation
